website/src/data/gallery.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "title": "AutoGen Playground",
```

Comment:
It should be groupchat playground. @thinkall please double check.

Reply:
Yes, I've renamed it to [Microsoft AutoGen Group Chat Playground](https://thinkall-autogen-playground.hf.space/app)

Reply:
resolved